From 21c38e366f7bc1497b4bd113e4f8564ef6ccbc17 Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Sun, 29 Jul 2018 14:18:23 -0500
Subject: Disable eieio-test-37-obsolete-name-in-constructor

The eieio-test-37-obsolete-name-in-constructor test has been disabled.

Its behavior is currently unpredictable.

Bug: https://debbugs.gnu.org/24503
Bug-Debian: http://bugs.debian.org/879020
Forwarded: not-needed
---
 test/automated/eieio-tests.el | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/automated/eieio-tests.el b/test/automated/eieio-tests.el
index 994e987b703..650e7696612 100644
--- a/test/automated/eieio-tests.el
+++ b/test/automated/eieio-tests.el
@@ -892,8 +892,8 @@ eieio--testing
 (defmethod constructor :static ((_x eieio--testing) newname &rest _args)
   (list newname 2))
 
-(ert-deftest eieio-test-37-obsolete-name-in-constructor ()
-  (should (equal (eieio--testing "toto") '("toto" 2))))
+;; (ert-deftest eieio-test-37-obsolete-name-in-constructor ()
+;;   (should (equal (eieio--testing "toto") '("toto" 2))))
 
 (provide 'eieio-tests)
 
